Gems/Atom/Bootstrap/Code/Source/BootstrapSystemComponent.cpp

```diff
@@ -45,6 +46,9 @@
 #include <BootstrapSystemComponent_Traits_Platform.h>
 
 AZ_CVAR(AZ::CVarFixedString, r_default_pipeline_name, AZ_TRAIT_BOOTSTRAPSYSTEMCOMPONENT_PIPELINE_NAME, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "Default Render pipeline name");
+AZ_CVAR(uint32_t, r_width, 1920, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "Starting window width in pixels.");
+AZ_CVAR(uint32_t, r_height, 1080, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "Starting window height in pixels.");
+AZ_CVAR(uint32_t, r_fullscreen, false, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "Starting fullscreen state.");
```

Comment:
It seems more appropriate to use setting registry for these so they can be put in .setreg files. And more specifically platform dependent setting registry (read with `AZ::SettingsRegistryMergeUtils::PlatformGet`), so different values per platform could be specified.

These CVars will appear on the console and could be modified at runtime, but they won't make any difference.

Reply:
Actually `PlatformGet` should not be used.
That was added just for porting over the bootstrap.cfg settings.

The recommended way to specify platform dependent settings registry settings is by adding a `Platform/<platform-name>` folder underneath the `Registry` directory that is being used (https://www.o3de.org/docs/user-guide/settings/developer-documentation/#merge-settings-files)

Reply:
Thanks for the clarification @lumberyard-employee-dm. I wasn't away of that recommendation.

Reply:
I understand your concern, and the problem we have is that there's no unified way at the moment. There are things in CVars, others in Registry settings, others in CFG files, etc. I'll leave it to the engineers to eventually clean that up. For now, this PR enables screen resolution setup while minimizing verbosity.